e2e-tests/custom-users-roles-sharded/conf/secrets.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
I can't understand why you need to have this secret. You even do not use it.

Reply:
Ah.. copy-paste thing.